packages/expo-gl-cpp/cpp/EXGL_CPP.podspec

```diff
@@ -14,16 +14,16 @@ Pod::Spec.new do |s|
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.source_files   = '**/*.{h,c,cpp,mm}'
   s.preserve_paths = '**/*.{h,c,cpp,mm}'
-  s.exclude_files  = 'UEXGL.*'
+  s.exclude_files  = '**/{UEXGL,EXGLContext,EXGLInstallConstants,EXGLInstallMethods,EXGLNativeMethods}*'
   s.requires_arc   = true
   
   s.pod_target_xcconfig = {
     'CLANG_WARN_COMMA' => 'NO',
     'CLANG_WARN_UNGUARDED_AVAILABILITY' => 'NO'
   }
   
-  s.subspec 'UEXGL' do |ss|
-    ss.source_files = 'UEXGL.*'
+  s.subspec 'EXGLContext' do |ss|
+    ss.source_files = '**/{UEXGL,EXGLContext,EXGLInstallConstants,EXGLInstallMethods,EXGLNativeMethods}*'
```

Comment:
Would it work if we completely get rid of this subspec and just use `s.compiler_flags = '-x objective-c++'` in the root spec? (without `exclude_files` ofc)